pkg/executables/builder.go

```diff
@@ -123,3 +134,18 @@ func NewLocalExecutableBuilder() *ExecutableBuilder {
 func DefaultEksaImage() string {
 	return defaultEksaImage
 }
+
+type Closer func(ctx context.Context) error
```

Comment:
Also does this need to be exported too?

Reply:
It's being returned by `NewExecutableBuilder`